### DISCUSSION TITLE
Typescript beta testing

### DISCUSSION THREAD

**Discussion by pomfrida** [equinor/design-system#867] [Code]
👋 
### Intro:
Let us know in this thread if you've found bugs or usability issues in our Typescript beta version. 
Any suggestions to fix or improve is also very much appreciated. Thanks! ✌️

### Install:
You can find the Typescript beta using the next tag:
* ` @equinor/eds-core-react@next` 
* `@equinor/eds-icons@next`
* ` @equinor/eds-tokens@next`

### Notes: 
* If you have declared a type module for `@equinor/eds-core-react` remember to remove this now. 
* If VSCode complains about missing declarations, try restarting VSCode
### Known issues:
* Compound components have inconsistent behaviour with types and intellisense in your IDE

**Comment by cbuv** [comment]
Upgrade was pretty straightforward. 
Tested on
````
    "react": "^17.0.1",
    "react-dom": "^17.0.1",
    "typescript": "~4.0.2",
    "@equinor/eds-core-react": "^0.7.0-beta.3",
    "@equinor/eds-icons": "^0.5.0-beta.1",
    "@equinor/eds-tokens": "^0.5.1-beta.1",
````

Textfield deprecated property errorMsg, using helperText instead in combination with variant. 

Some unexpected changes: 
* TextField has no longer any rows property (when multiline is true)
* Icon has no longer variant property. 
![image](https://user-images.githubusercontent.com/48285335/98955052-c2988280-24fe-11eb-908b-647628760e2b.png)

**Reply by pomfrida** [reply]
Thanks @cbuv for the quick response! I have a question about:

> _Icon has no longer variant property._

Icon never had a `variant` prop, did you mean something else perhaps? 🤔

**Reply by cbuv** [reply]
You're right, its was just a dead property on our side.
